sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -739,7 +739,7 @@ static void alert_handler_config(void) {
       .classes = classes,
       .class_configs = class_config,
       .classes_len = ARRAYSIZE(class_config),
-      .ping_timeout = 0,
+      .ping_timeout = 256,
```

Comment:
i remember Michael mentioned a default number for ping timeout. If you already confirmed with him, lgtm :)

Reply:
Yes this looks good.